internal/api/features/topics.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ */
+
+package features
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+//go:embed _topics.json
+var topicsData string
+
+func GetTopics() TopicInfo {
+	var tp TopicInfo
+	_ = json.Unmarshal([]byte(topicsData), &tp)
+	return tp
+}
+
+func BuildFilterConsistAny(key string, values []string) (string, error) {
+	if strings.TrimSpace(key) == "" {
+		return "", fmt.Errorf("empty key: %q", key)
+	}
+	return fmt.Sprintf("%s: consistAny {%s}", key, strings.Join(values, ",")), nil
+}
```

Comment:
this is fine, but doesn't belong in the `api` package. The `api` package must include the API types definition and the most essential support code (e.g. validation, deepcopy...)

Reply:
If the problem is testing, or making testing easier, we can still solve the problem convenientely using e2e tests for the `mkginkgolabelfilter` binary. The binary is going to be very easy and simple to test (no options! no state! can run locally!) so e2e tests will be fast and convenient.

Reply:
makes sense, the tool e2e tests cover that indeed, which is the core of the tool. I'll remove this from here